### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Lochipi**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by emmett23** [comment]
Nice one

**Review by ghost — COMMENTED** [review]
This is amazing, quality code

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Lochipi) [inline index.html]: cool, would you mind using icons, import it and use fontawesome

Comment (Lochipi) [inline index.html]:
```suggestion
<h2 hidden>The game is over! you lost, refresh and try again 🧑‍💻 🤖!</h2>
```

**Pull request merged**